.github/workflows/rust_ci.yaml

```diff
@@ -61,6 +62,16 @@ jobs:
         run: |
           cargo make --profile development-linux-x86_64 appflowy-core-dev
 
+      - name: Setup NDK
```

Comment:
Same as the previous one.

Reply:
The NDK helps compile rust libraries to readable code for Android. So it is needed.